SDL/CocoaBarItems.mm

```diff
@@ -51,6 +51,16 @@ void initializeOSXExtras() {
     [[BarItemsManager sharedInstance] setupAppBarItems];
 }
 
+void OSXShowInFinder(const char *path) {
+    NSURL *url = [NSURL fileURLWithPath:@(path)];
+    [NSWorkspace.sharedWorkspace activateFileViewerSelectingURLs:@[url]];
```

Comment:
this is fine since it's following the tab-ness of the rest of the file (which we should change at some point, but meh)